serverless/sql-databases/api-cli/import-data-to-serverless-sql-databases.mdx

```diff
@@ -0,0 +1,528 @@
+---
+meta:
+  title: Importing data into Serverless SQL Databases
+  description: This page explains how to import your data into Serverless SQL Databases from another database or backup files
+content:
+  h1: Importing data into Serverless SQL Database
+  paragraph: This page explains how to import your data into Serverless SQL Databases from another database or backup files
+dates:
+  validation: 2024-06-24
+  posted: 2024-06-24
+---
+
+Serverless SQL Databases offers near-complete PostgreSQL compatibility, and allows you to import your data from any PostgreSQL-compatible database.
+
+<Message type="important">
+The methods listed below are best suited for database sizes below 100 GB (approx.). Above this size, we recommend you use dedicated tooling to perform this operation, as your local storage size and bandwidth will create bottlenecks, and the operation may last several hours.
+</Message>
+
+The import procedure depends on your data source:
+    - [Importing data from an existing PostgreSQL database](#importing-data-from-an-existing-postgresql-database)
+      - [Using pg_dump](#using-pgdump)
+      - [Using pgAdmin](#using-pgadmin)
+      - [Using Airbyte](#using-airbyte)
+      - [Using Fivetran](#using-fivetran)
+      - [Using Meltano](#using-meltano)
+      - [Using Apache NiFi](#using-apache-nifi)
+    - [Importing data from an existing PostgreSQL backup](#importing-data-from-an-existing-postgresql-backup)
+    - [Importing data from a file (.csv, .txt, etc.)](#importing-data-from-a-file-csv-txt-etc)
+
+## Importing data from an existing PostgreSQL database
+
+### Using pg_dump
+
+#### Before you start
+
+To complete the actions presented below, you must have:
```

Comment:
Does it also need the standard requirements macro? (Created a SCW account, IAM rights etc)